ng-dev/pr/rebase/index.ts

```diff
@@ -85,13 +85,16 @@ export async function rebasePr(prNumber: number, githubToken: string) {
   }
 
   try {
+    // Fetches are done with --depth=500 increase the likelihood of finding a common ancestor when
```

Comment:
Does using `--unshallow` work as well?

Reply:
That was what I thought too, but I tried it and got: `fatal: --unshallow on a complete repository does not make sense`

Reply:
Oh weird. Especially since the `git man` also talks about the complete repository case. good to have this captured here in the PR though. We thought about it 😄